### PR TITLE
modify loggers

### PR DESCRIPTION
**Pull request by SaxenaSim**

### CODE REVIEW

File: logging_config.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ handlers:
   file_handler:
     class: logging.FileHandler
     formatter: simple
-    filename: /home/user1/Desktop/cypherQueryGeneration/logs/my_logfile.log
+    filename: /home/user1/Desktop/neo4jGraphDatabase/logs/my_logfile.log
```
Comment (prarshah1) [inline logging_config.yaml:27]: if possible, try to keep file names relative to project structure, create folders in resources/ if necessary

File: requirements.txt
Comment (prarshah1) [inline requirements.txt:1]: have we used any other libraries? like langchain or streamlit?

File: src/loadCSVfiles.py
Comment (prarshah1) [inline src/loadCSVfiles.py:1]: what is the difference between load csv and populate data?

File: src/log.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        print(config_logging_path)
+
+        with open(config_logging_path, 'r') as f:
+            config = yaml.safe_load(f.read())
+            logging.config.dictConfig(config)
+
+        cls._logger = logging.getLogger(__name__)
+
+    @classmethod
+    def getlogger(cls):
```
Comment (prarshah1) [inline src/log.py:22]: wont it always be none?

File: src/loadCSVfiles.py [hunk 3]
```diff
@@ -61,11 +51,12 @@ def load_csv_data(tx) -> None:
         
         # Open a session and execute the CSV loading queries within a transaction
         with driver.session() as session:
-            logger.info("Before loading files")
+            print("session opened")
             session.execute_write(CSVLoader.load_csv_data)
+            print("method called")
```
Comment (prarshah1) [inline src/loadCSVfiles.py]: replace print

File: src/log.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import os , yaml
+import logging , logging.config
+
+class Logger:
+    _logger = None
+    
+    @classmethod
+    def createlogger(cls):
```
Comment (prarshah1) [inline src/log.py:8]: use camel case

File: src/textToCypherQuery.py [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    # Get the Neo4j driver and execute the query   
+    def _run(self,query):
+        driver = Driver.get_driver()
+        with driver.session() as session:
+            result = session.run(query)
+            records = list(result)  
+        return records
+    
+# Main class to handle text to Cypher query conversion and execution
+class textToCypher:
```
Comment (prarshah1) [inline src/textToCypherQuery.py:26]: class name should start with a capital letter, install some lint tool for validation

File: src/textToCypherQuery.py
Comment (prarshah1) [inline src/textToCypherQuery.py:1]: The official convention is to use all lower case for file names  as filenames in python as case sensitive, so camel case is preferred

File: src/textToCypherQuery.py (hunk 5 above)
Comment (prarshah1) [reply src/textToCypherQuery.py:26]: test cases for this file?

File: src/textToCypherQuery.py [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+            agent=execution_agent
+        )
+        self.logger.info("Tasks created successfully")
+        return task , execution_task
+     
+    # Method to run the tasks using agents and tools   
+    def run(self):
+        self.logger.info("Starting run method to get cypher query")
+        obj_agent, execution_agent = self.create_agents()
+        task, execution_task = self.create_tasks(obj_agent, execution_agent)
```
Comment (prarshah1) [inline src/textToCypherQuery.py]: if possible, add more descriptive names like task name = { purpose _ task }

File: src/textToCypherQuery.py [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+
+        #result = crew.kickoff(inputs={"text_input": text_input})
+        #result = crew.kickoff(inputs={"text_input":"Give me the list of consumers who has business as Arlin Toomey."})
+        
+if __name__ == "__main__":
+            
+    #text_input = "Give me the list of consumers who has business as Arlin Toomey."
+    
+    text_to_cypher = textToCypher()
+    text_to_cypher.run()
```
Comment (prarshah1) [inline src/textToCypherQuery.py:137]: where is the input natural language text?

File: requirements.txt
Comment (SaxenaSim) [reply requirements.txt:1]: no , for now only crewai is been used

File: src/loadCSVfiles.py
Comment (SaxenaSim) [reply src/loadCSVfiles.py:1]: deleted the populateData.py file as it was not needed

File: src/loadCSVfiles.py (hunk 3 above)
Comment (SaxenaSim) [reply src/loadCSVfiles.py]: removed the print statements

File: src/textToCypherQuery.py (hunk 7 above)
Comment (SaxenaSim) [reply src/textToCypherQuery.py:137]: added the input as it was showing error initially

File: src/textToCypherQuery.py (hunk 6 above)
Comment (SaxenaSim) [reply src/textToCypherQuery.py]: changes the name of tasks and agents

File: src/textToCypherQuery.py
Comment (SaxenaSim) [reply src/textToCypherQuery.py:1]: changed the name of python scripts according to camelCase

File: src/textToCypherQuery.py (hunk 5 above)
Comment (SaxenaSim) [reply src/textToCypherQuery.py:26]: working on test cases right now

File: src/log.py (hunk 4 above)
Comment (SaxenaSim) [reply src/log.py:8]: changed the logger

**Pull request merged**